geomstats/geometry/discrete_curves.py

```diff
@@ -326,6 +331,50 @@ def square_root_velocity_inverse(self, srv, starting_point):
 
         return curve
 
+    def aux_differential_square_root_velocity(self, tangent_vec, curve):
```

Comment:
Thanks for the renaming! I think DeepSource is complaining about the docstring, adding the Parameters and their description here would solve it.